packages/dd-trace/src/tracer_metadata.js

```diff
@@ -0,0 +1,25 @@
+'use strict'
+
+// Load binding first to not import other modules if it throws
+const libdatadog = require('@datadog/libdatadog')
+const tracerVersion = require('../../../package.json').version
```

Comment:
```suggestion
const tracerVersion = require('../../../version')
```

That actually exposes the information as well and it's less work to read it than the package.json :)

We could just export the raw version from there as well. That way there is no combining needed either.